src/main/java/seedu/cafectrl/CafeCtrl.java

```diff
@@ -0,0 +1,53 @@
+package seedu.cafectrl;
+
+import seedu.cafectrl.command.Command;
+import seedu.cafectrl.command.ExitCommand;
+import seedu.cafectrl.data.Menu;
+import seedu.cafectrl.parser.Parser;
+import seedu.cafectrl.ui.Ui;
+
+public class CafeCtrl {
+    private final Ui ui;
+    private final Menu menu;
+    private Command command;
+
+    /**
+     * Private constructor for the CafeCtrl class, used for initializing the user interface and menu list.
```

Comment:
Perhaps you meant public constructor?

Reply:
Sorry I was suppose to declare the CafeCtrl constructor as private... I'm going to change that now thank you!